app/models/framework/definition/RM3767.rb

```diff
@@ -9,38 +9,38 @@ class RM3767 < Base
       class Invoice < Sheet
         total_value_field 'Total Cost (ex VAT)'
 
-        field 'Lot Number', :string, exports_to: 'LotNumber'
-        field 'Customer URN', :integer, exports_to: 'CustomerURN'
-        field 'Tyre Specification ', :string, exports_to: 'ProductCode'
+        field 'Lot Number', :string, exports_to: 'LotNumber', presence: true, inclusion: { in: %w[1 2] }
+        field 'Customer URN', :integer, exports_to: 'CustomerURN', urn: true
+        # field 'Tyre Specification', :string, exports_to: 'ProductCode', presence: true
         field 'Vehicle Category', :string, exports_to: 'ProductSubClass'
         field 'Associated Service', :string, exports_to: 'ProductDescription'
-        field 'Tyre Width', :decimal
-        field 'Aspect Ratio', :integer
-        field 'Rim Diameter', :decimal
-        field 'Load Capacity', :integer
+        field 'Tyre Width', :string
+        field 'Aspect Ratio', :string
+        field 'Rim Diameter', :string
+        field 'Load Capacity', :string
```

Comment:
Same here, no validation?

Reply:
These are optional string values, so they don't get validated.

Reply:
So presence doesn't get checked, but are we happy if someone submits some random text in here? It's just odd it was coerced into a number before, and now we're happy with *any* string.